modules/twinkleimage.js

```diff
@@ -198,7 +199,8 @@ Twinkle.image.callback.evaluate = function twinkleimageCallbackEvaluate(event) {
 		reason = event.target.reason.value;
 	}
 	if (event.target.replacement) {
-		replacement = event.target.replacement.value.replace(/^\s*(Image|File):/i, '');
+		replacement = event.target.replacement.value;
+		replacement = replacement.indexOf('Image:') === 0 || replacement.indexOf('File:') === 0 ? replacement : 'File:' + replacement;
```

Comment:
```suggestion
		replacement = /^\s*(Image|File):/i.test(replacement) ? replacement : 'File:' + replacement;
```

Reply:
Requires some handling of empty or whitespace input a la `speedy`.